test/perf-kinesis/mzcompose.py

```diff
@@ -12,22 +12,24 @@
 from materialize.mzcompose import Composition, Service
 from materialize.mzcompose.services import Materialized, PrometheusSQLExporter
 
+PerfKinesis = Service(
+    name="perf-kinesis",
+    config={
+        "mzbuild": "perf-kinesis",
+        "environment": [
+            "RUST_LOG=perf-kinesis=debug,info",
+            "AWS_ACCESS_KEY_ID",
+            "AWS_DEFAULT_REGION=us-east-2",
+            "AWS_SECRET_ACCESS_KEY",
+            "AWS_SESSION_TOKEN",
+        ],
+    },
+)
+
```

Comment:
Ah, so this is not idiomatic for a mzcompose.py, and I would really prefer not to have any of these out-of-line definitions. (Any existing out-of-line definitions in other compositions are style bugs that I've been meaning to correct.)

Here is my argument:

* There is only the smallest of benefits of referring to `PerfKinesis.name` rather than "perf-kinesis": it makes typos a pycheck error rather than a runtime error. But it makes the callsites much noisier than just using the string literal. I think the readability of the callsite is more important.
* Extracting out-of-line definitions for some services, but not all, is the worst possible case, because now sometimes you see `"materialized"` but sometimes `PerfKinesis.name`. The disparity is confusing.
* If we  extract out-of-line definitions for each service, the result is lots of mandatory extra verbosity:
```py
Materialized = Materialized()
PrometheusSQLExporter = PrometheusSQLExporter()
# ...
SERVICES = [
  Materialized,
  PrometheusSQLExporter,
]
```

If we find that using strings causing actual problems in practice, then we could start thinking about how to recover type safety in a way that generalizes to all defined services without a verbosity tax. But I'm really unconvinced that it will ever be a pressing problem in practice.